README.md

```diff
@@ -31,6 +31,11 @@ An `$ExpectType` directive tests the type of the expression on the next line. Th
     npm install -g typings-checker
     typings-checker your-test.ts
 
+## options
+
+* `--noLines` (`-l`): hide line numbers in output -- useful if you don't want these clogging diff logs when you add or remove a line.
```

Comment:
Can you explain the use case for this a little more? I'd expect most users of typings-checker would just care about the exit code. As an analogy, the `tsc` command doesn't have an option to suppress line numbers. Ideally we'd just have a single output format that's good for everyone.